proposals/description/tip_121b.ts

```diff
@@ -0,0 +1,157 @@
+      arguments: (addresses) => [
+        addresses.daiHoldingPCVDeposit, // pcvDeposit
+        addresses.lusd, // token
+        addresses.lusdHoldingPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: `Move all LUSD from daiHoldingPCVDeposit to lusdHoldingPCVDeposit.`
+    },
+    // 2. Withdraw WETH and DAI from WETH->DAI LBP swapper, move to Lido
+    {
+      target: 'ethToDaiLBPSwapper',
+      values: '0',
+      method: 'exitPool(address)',
+      arguments: (addresses) => [addresses.daiHoldingPCVDeposit],
+      description: 'Move all DAI and WETH to daiHoldingPCVDeposit'
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioERC20(address,address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.daiHoldingPCVDeposit, // pcvDeposit
+        addresses.weth, // token
+        addresses.wethHoldingPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: `Move all WETH from daiHoldingPCVDeposit to wethHoldingPCVDeposit`
+    },
+    {
+      target: 'ratioPCVControllerV2',
+      values: '0',
+      method: 'withdrawRatioUnwrapWETH(address,address,uint256)',
+      arguments: (addresses) => [
+        addresses.wethHoldingPCVDeposit, // pcvDeposit
+        addresses.ethLidoPCVDeposit, // to
+        '10000' // basisPoints, 100%
+      ],
+      description: `Move all ETH from wethHoldingPCVDeposit to ethLidoPCVDeposit to deposit()`
+    },
+    {
+      target: 'ethLidoPCVDeposit',
+      values: '0',
+      method: 'deposit()',
+      arguments: (addresses) => [],
+      description: `Deposit ETH in Lido`
+    },
+
+    // 3. Cleanup FEI/TRIBE on TC Timelock from the Rari Infra team clawback
+    {
+      target: 'tribe',
+      values: '0',
+      method: 'transferFrom(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribalCouncilTimelock, addresses.core, TC_TRIBE_BALANCE],
+      description: 'Move all 2.7M TRIBE from TC Timelock to DAO Treasury'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'transferFrom(address,address,uint256)',
+      arguments: (addresses) => [addresses.tribalCouncilTimelock, addresses.feiDAOTimelock, TC_FEI_BALANCE],
+      description: 'Move all 2.7M FEI from TC Timelock to DAO Timelock'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'burn(uint256)',
+      arguments: (addresses) => [TC_FEI_BALANCE],
+      description: 'Burn 2.7M FEI from the DAO Timelock'
+    },
+    // 4. Update CR oracle to use stETH oracle for stETH rather than ETH oracle
+    {
+      target: 'collateralizationOracle',
+      values: '0',
+      method: 'setOracle(address,address)',
+      arguments: (addresses) => [addresses.weth, addresses.chainlinkStEthUsdOracleWrapper],
+      description: 'Update the ETH oracle in the CR oracle, to the stETH oracle as remaining ETH will be stETH'
+    },
```

Comment:
what is the price of this oracle vs ETH oracle

Reply:
stETH: $1531.92
ETH: $1578.02

Reply:
as of block pinned in e2e tests